.eslintignore

```diff
@@ -1,2 +1,5 @@
 node_modules
-dist
\ No newline at end of file
+dist
+src/markup-components/component-wrapper.tsx
+src/markup-components/components.ts
+src/markup-components/express.ts
```

Comment:
But why can't the files be linted?

Reply:
they can, but they require some changes. I don't want to touch them right now since I don't have tests in elements to verify that the changes worked.